modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/MultiselectPicklistFDSPropsTransformer.tsx

```diff
@@ -11,6 +14,25 @@ interface MultiselectPicklistDataRendererProps {
 	}[];
 }
 
+interface SourceDataRenderer {
+	value: boolean;
+}
+
+function SourceDataRenderer({value}: SourceDataRenderer) {
```

Comment:
Can we have a generic SourceDataRenderer? this way we can avoid duplication of code like these in [ObjectFieldSourceDataRenderer](https://github.com/liferay/liferay-portal/blob/0d251126abe828f08269a278d2b81b2b83111a1d/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/ObjectFieldsFDSPropsTransformer.tsx#L13-L26) and [objectFieldSourceDataRenderer](https://github.com/liferay/liferay-portal/blob/0d251126abe828f08269a278d2b81b2b83111a1d/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Fields.tsx#L105-L118)
We will also need this renderer for the next stories (Actions, Validations and so on)